### ISSUE TITLE
Translating app into other languages

### ISSUE THREAD

**Issue by rakleed** [linearmouse/linearmouse#66]
Could you please provide the community with the ability to translate the app into other languages?

P.S. I can help with translation into Russian.

**Comment by lujjjh** [comment]
Sure! Contributions are always welcome!

It's quite simple to contribute a translation, even without Xcode:

1. Copy `LinearMouse/zh-Hans.lproj/Localizable.strings` to `LinearMouse/[lang].lproj/Localizable.strings`. (`[lang]` should be replaced with the two-letter [ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). For example, `ru` for Russian.)
2. Edit the strings file with your faviorite text editor.
3. Submit a pull request and I'll take care of the rest.

**Comment by rakleed** [comment]
@lujjjh what does line https://github.com/linearmouse/linearmouse/blob/09b49bb90d324c1d8d87c993c4e4f133b9637121/LinearMouse/zh-Hans.lproj/Localizable.strings#L47 mean? I couldn't find it when using the app.

-----

And why don't you update the https://github.com/linearmouse/linearmouse/blob/main/LinearMouse/en.lproj/Localizable.strings file?

**Comment by lujjjh** [comment]
> [...] you tweaked the preferences UI slightly, also changing the "Speed factor" value to "to", but for some reason you left the "Speed factor" line in the code.

Oops! I forgot to remove this line. Please ignore it.

> And why don't you update the https://github.com/linearmouse/linearmouse/blob/main/LinearMouse/en.lproj/Localizable.strings file?

This file may also be deleted since the development language is English.

**Comment by rakleed** [comment]
@lujjjh can you add to the README information on how to add or correct a translation? To make it easier for those who want to help with the translation.

**Comment by lujjjh** [comment]
> @lujjjh can you add to the README information on how to add or correct a translation? To make it easier for those who want to help with the translation.

Yes, I added a guide in [CONTRIBUTING.md](https://github.com/linearmouse/linearmouse/blob/main/CONTRIBUTING.md#contributing-localizations).

I also created a project in [Crowdin](https://crowdin.com/project/linearmouse). I'm hoping it will make contributing and maintaining translations more convenient.
